<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Invite</class>
 <widget class="QWidget" name="Invite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>915</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_ajouter_invite">
    <widget class="QLabel" name="back_p1_invite">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>911</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
image: url(:/img/img/4530621.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>391</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
border-radius:60px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/img/user-plus-solid (1).svg);
border: 3px solid #fa9746;
border-radius:50%;
padding:10px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nomInv">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid #fa9746;
background-color:transparent;
color:black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prenomInv">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid #fa9746;
background-color:transparent;
color:black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajoute_invite">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:2px solid #fa9746;
border-radius:25px;
color:grey;
background-color:transparent;
</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_telInv">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>240</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid #fa9746;
background-color:transparent;
color:black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Téléphone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_emailInv">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid #fa9746;
background-color:transparent;
color:black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <zorder>lineEdit_nomInv</zorder>
     <zorder>label_5</zorder>
     <zorder>lineEdit_prenomInv</zorder>
     <zorder>pushButton_ajoute_invite</zorder>
     <zorder>lineEdit_telInv</zorder>
     <zorder>lineEdit_emailInv</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="page_afficher_invite">
    <widget class="QLabel" name="back_p2_invite">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>911</width>
       <height>451</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableView" name="tableInvite">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>621</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_search_invite">
     <property name="geometry">
      <rect>
       <x>814</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/search-solid (1).svg);
background-color:transparent;
border-radius:30px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>602</x>
       <y>10</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid  #fa9746;
margin:0 10px;
background:none;</string>
     </property>
     <property name="placeholderText">
      <string>Recherche</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid  #fa9746;
color: #fa9746;
margin:0 10px;
background:none;</string>
     </property>
     <property name="text">
      <string>Filtrer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_filter_invite">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/filter-solid (1).svg);
background-color:transparent;
border-radius:30px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_print_invite">
     <property name="geometry">
      <rect>
       <x>814</x>
       <y>410</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/print-solid.svg);
background-color:transparent;
border-radius:30px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>410</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid  #fa9746;
color: #fa9746;
margin:0 10px;
background:none;</string>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_modifier_invite">
    <widget class="QLabel" name="back_p3_invite">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>911</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/robots-human-candidates-waiting.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>391</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
border-radius:60px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/img/user-edit-solid.svg);
border: 3px solid #fa9746;
border-radius:50%;
padding:10px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Nom_Mod_inv">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid #fa9746;
background-color:transparent;
color:grey;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Prenom_Mod_inv">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid #fa9746;
background-color:transparent;
color:grey;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ModInv">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:2px solid #fa9746;
border-radius:25px;
background-color:transparent;
color:grey;
</string>
      </property>
      <property name="text">
       <string>Enregistrer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Tel_Mod_Inv">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid #fa9746;
background-color:transparent;
color:grey;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Téléphone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Email_Mod_Inv">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid #fa9746;
background-color:transparent;
color:grey;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_suppr_invite">
    <widget class="QLabel" name="back_p4_invite">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>911</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/img/img/dilemma-with-woman.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>371</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
border-radius:60px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="lineEdit_suppr_invite">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:none;
border-bottom:2px solid #c25711;
background-color:transparent;
color:grey;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Téléphone</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:18px;
font-family:Arial;
border:2px solid #c25711;
border-radius:25px;
background-color:transparent;
color:grey;
</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/img/user-minus-solid (1).svg);
border: 3px solid #c25711;
border-radius:50%;
padding:10px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Home">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>16</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_Ajouter">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ajouter un invité&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Afficher">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Modifier">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Suppr">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-family:Arial;
border:none;

color:#fc7a2e;
margin:0 10px;
background-color:transparent;</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
